sdk-docs/testing-iris-apps/mock-out-runtime-hooks.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Mocking out runtime hooks
+category: 61fcd8e1a448f5004215317c
+parentDocSlug: testing-iris-apps
+---
+
+> 🚧 Beta
+> 
+> This is a beta version and subject to change without notice. Pricing, terms, conditions and availability may change in the final version.
+
+
+If you need to mock out runtimes you can do it like this, lets say you are using our hook `useAssetRuntime()` in your Iris App extension.
```

Comment:
Same as above regarding talesprog that doesn't come off well in text.
Tip: my process is to write something in my own words without thinking too much, I paste it into chatgpt and tell it to improve my grammar and language, I then modify it to make it sound less clinical :)